CRM/Utils/System/Base.php

```diff
@@ -1004,4 +1004,12 @@ public function viewsExists() {
   public function prePostRedirect() {
   }
 
+  /**
+   * Return the CMS-specific url for its permissions page
+   * @return array
+   */
+  public function getCMSPermissionsUrlParams() {
+    return NULL;
```

Comment:
@demeritcowboy should this be an empty array instead of NULL?

Reply:
Thanks for looking. Yep that would be more consistent. Have updated.